### ISSUE TITLE
[Bug] Section endpoint does not work

### ISSUE THREAD

**Issue by NicKravchenko** [SoftwyRD/university-api#37]
# Bug

I was testing creation of selection and then of section. 
Endpoint ```/api/selections/<uuid:selection_id>/subjects/```  returns the same result from Postman and Swager for GET and POST methods: 
```
{
    "status": "error",
    "message": "There was an error trying to get the subjects."
}
```
Above you can see images of error:

![Image](https://user-images.githubusercontent.com/89710564/219549794-995ec877-9d64-4cc0-b827-f57add67cd2d.png)

![Image](https://user-images.githubusercontent.com/89710564/219550326-ecc3d2a6-3209-4787-9846-e5319e9b5d52.png)

Hope it will be fixed  asap.

**Comment by OmarNunezG** [comment]
Hi @NicKravchenko 

I tried to replicate the error, but I got a **200 status code**. I tested in 1b0a43f with both swagger and postman.

This are the responses:

- Postman
![Postman](https://user-images.githubusercontent.com/69877860/219661746-f93fe65e-b6bd-4273-9ddf-413b3fafa1f2.png)

- Swagger
![Swagger](https://user-images.githubusercontent.com/69877860/219662081-042d9d75-37e6-46e7-a594-916f30a4bc0b.png)

Could you trace the error?

Let me know what the issue was when you figure it out!

**Comment by NicKravchenko** [comment]
Hmm, my bad, but post didnt work

**Comment by OmarNunezG** [comment]
There was an error while trying to reverse the new selection's subject url.

Fixed on e478c6c.
